### PR TITLE
no network notification

### PR DESCRIPTION
**Pull request by yimenglovesai**
https://github.com/Loopring/loopr-ios/issues/42
screen shot 2018-04-14 at 8 24 31 pm

### CODE REVIEW

**Comment by xiaowheat** [comment]
@yimenglovesai Could you add a screenshot and a description?

File: loopr-ios/New Group/NetworkingManager.swift [hunk 1]
```diff
@@ -0,0 +1,178 @@
+//
+//  NetworkingManager.swift
```
Comment (xiaowheat) [inline loopr-ios/New Group/NetworkingManager.swift]: Please move this file to Utils folder.

File: loopr-ios/New Group/NetworkingManager.swift [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+/// Defines the various connection types detected by reachability flags.
+///
+/// - ethernetOrWiFi: The connection type is either over Ethernet or WiFi.
+/// - wwan:           The connection type is a WWAN connection.
+public enum ConnectionType {
+    case ethernetOrWiFi
+    case wwan
+}
+
+class NetworkingManager {
```
Comment (xiaowheat) [inline loopr-ios/New Group/NetworkingManager.swift]: Please rename NetworkingManager to NetworkReachabilityManager

Comment (xiaowheat) [reply loopr-ios/New Group/NetworkingManager.swift]: @yimenglovesai

**Comment by yimenglovesai** [comment]
The original file was in Utils group, but like DateUtils, it's somehow located in New Group folder.

**Pull request merged**